### PR TITLE
config change for velox worker for memory allocation error

### PR DESCRIPTION
**Pull request by minhancao**
config add for velox worker for memory allocation error
`system-memory-gb=2`

`VeloxRuntimeError: Could not allocate working memory mmap failed with Cannot allocate memory for sizeClass 1`

### CODE REVIEW

File: docker-compose-native/prestissimo/workers/1/config.properties [hunk 1]
```diff
@@ -2,3 +2,4 @@ coordinator=false
 http-server.http.port=8080
 discovery.uri=http://172.19.0.4:8080
```
Comment (wanglinsong) [inline docker-compose-native/prestissimo/workers/1/config.properties:3]: Can you use the service name?

**Review by wanglinsong — CHANGES_REQUESTED** [review]

File: docker-compose-native/prestissimo/workers/1/config.properties (hunk 1 above)
Comment (minhancao) [reply docker-compose-native/prestissimo/workers/1/config.properties:3]: @wanglinsong I'm not sure what you mean

Comment (wanglinsong) [reply docker-compose-native/prestissimo/workers/1/config.properties:3]: Use service name instead of ip address to reference coordinator.

Comment (wanglinsong) [reply docker-compose-native/prestissimo/workers/1/config.properties:3]: Like in https://github.ibm.com/lakehouse/presto-performance/blob/main/presto-deploy-cluster/clusters/large/workers/config.properties

**Review by wanglinsong — APPROVED** [review]

**Pull request merged**